Documentation/gettingstarted/bgp.rst

```diff
@@ -60,7 +60,7 @@ Here are the required Helm values:
 
 .. parsed-literal::
 
-   helm install cilium |CHART_RELEASE| --set bgp.enabled=true --set bgp.announce.loadbalancerIP=true
+   helm install cilium |CHART_RELEASE| --set bgp.enabled=true --set bgp.announce.loadbalancerIP=true --namespace kube-system
```

Comment:
Thanks! Would you mind moving the `--namespace` option before the config options, please?
Since the line is getting longer, we could even split it, to make it consistent with most other `helm` invocations in the Documentation:

```suggestion
   helm install cilium |CHART_RELEASE| \\
     --namespace kube-system \\
     --set bgp.enabled=true \\
     --set bgp.announce.loadbalancerIP=true
```

Reply:
Thanks for the review!
Force-pushed fixes.